### PR TITLE
Add between operator #56

### PR DESCRIPTION
**Pull request by Th3Mouk**
## Purpose
#56 

## Approach
Reading the doc and reading drivers in others langages 

## Research
r.minval and r.maxval doesn't work

### CODE REVIEW

File: src/Query/Operation/Between.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<?php
+
```
Comment (marcgeurts) [inline src/Query/Operation/Between.php]: declare strict type missing

**Comment by Th3Mouk** [comment]
Tests pass locally can't reproduce

File: src/Query/Operation/Between.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+     * @var string
+     */
+    private $options;
+
+    public function __construct(
+        RethinkInterface $rethink,
+        QueryInterface $query,
+        $min,
+        $max,
+        $options = null
```
Comment (tbolier) [inline src/Query/Operation/Between.php]: You can define the argument type as `array $options = null`.

File: src/Query/Table.php [hunk 3]
```diff
@@ -65,6 +66,20 @@ public function indexRename(string $oldValue, string $newValue): AbstractQuery
         return new IndexRename($this->rethink, $this, $oldValue, $newValue);
     }
 
+    public function between(
+        $min,
+        $max,
+        $options = null
```
Comment (tbolier) [inline src/Query/Table.php]: You can define the argument type as `array $options = null`.

**Comment by tbolier** [comment]
Can we create a new issue for `r.minval` and `r.maxval` so we have a reference for future?

**Comment by Th3Mouk** [comment]
Yes this constants doesn't work currently this must be an issue

**Comment by tbolier** [comment]
@Th3Mouk I am planning to merge this now, do you agree?

**Pull request merged**